dotCMS/src/main/java/com/dotcms/jitsu/EventLogWebInterceptor.java

```diff
@@ -98,18 +101,36 @@ public void doPost(final HttpServletRequest request,
             return;
         }
 
-        final String payload = IOUtils.toString(request.getReader());
+        final String requestPayload = IOUtils.toString(request.getReader());
         final String realIp = request.getRemoteAddr();
-        final JSONObject json = new JSONObject(payload);
+        final Map<String, Object> requestJsonPayload = JsonUtil.getJsonFromString(requestPayload);
+        final List<Map<String, String>> experiments = (List<Map<String, String>>) requestJsonPayload.get("experiments");
+
+        if (!UtilMethods.isSet(experiments)) {
+            return;
+        }
+
+        requestJsonPayload.remove("experiments");
+
+        final EventsPayload eventPayload = new EventsPayload(requestJsonPayload);
         final String mungedIp = (realIp.lastIndexOf(".") > -1)
                 ? realIp.substring(0, realIp.lastIndexOf(".")) + ".1"
                 : "0.0.0.0";
-        json.put("ip", mungedIp);
-        json.put("clusterId", ClusterFactory.getClusterId());
+        eventPayload.put("ip", mungedIp);
+        eventPayload.put("userLanguage", String.valueOf(
+                WebAPILocator.getLanguageWebAPI().getLanguage(request).getId()));
+        eventPayload.put("persona",
+                WebAPILocator.getPersonalizationWebAPI().getContainerPersonalization(request));
+        //eventPayload.put("clusterId", ClusterFactory.getClusterId());
```

Comment:
For some reasons when we set this clusterID we are getting an error in jitsu.
The error something like that it is not be able to ALTER the events table in clickhouse to add a new field call clusterid.

Also the events already has a cluster_id set, I don't know were we are setting this and if this is ok

Reply:
I am going to create a new issue to fix (if it is needed to fix) all this later

Reply:
Issue created https://github.com/dotCMS/core/issues/23834

Reply:
We have to have the cluster_id set in clickhouse. This is how we multi-tenant